src/lockfile/index.js

```diff
@@ -102,14 +102,14 @@ export default class Lockfile {
 
   static async fromDirectory(dir: string, reporter?: Reporter): Promise<Lockfile> {
     // read the manifest in this directory
-    const lockfileLoc = path.join(dir, constants.LOCKFILE_FILENAME);
+    const lockfileLoc = path.join(dir, LOCKFILE_FILENAME);
 
     let lockfile;
     let rawLockfile = '';
     let parseResult;
 
-    if (await fs.exists(lockfileLoc)) {
-      rawLockfile = await fs.readFile(lockfileLoc);
+    if (await exists(lockfileLoc)) {
```

Comment:
I'd rather keep this the old way to find all `fs.exists` calls easily in the future but I'm guessing this is to reduce output size of the bundle by limiting the imports?